packages/client/hmi-client/src/types/Model.ts

```diff
@@ -1,12 +1,7 @@
-import { PetriNet } from '@/utils/petri-net-validator';
 import { CONCEPT_FACETS_DISPLAY_NAME, CONCEPT_FACETS_FIELD } from './Concept';
 import { Filters } from './Filter';
 
-// FIXME: other model content types will be supported depending on
-//	Model.framework
-export type ModelContent = PetriNet;
-
-export type Model = {
+export type ConcreteModel<T> = {
```

Comment:
Should we name it `GenericModel` instead? What does _concrete_ means in this context?

Reply:
@YohannParis @shawnyama 

Sorry about the confusion, It is actually the reverse of what you are inferring, this is a specific model-type, as it takes in a typed-generic.

I do agree that concrete sounds weird. I offered from above that we should renamed ConcreteModel
- TypeModel<T> to indicate a model of specific type. e.g. TypedModel<PetriNet>, TypedModel<ABM> ..., and
- Model, as a "generic" model reference which is an alias for TypedModel<any>....to be used when you don't care about the underlying framework semantics, for example in facets.